sopel/irc.py

```diff
@@ -332,7 +318,10 @@ def _get_cnames(self, domain):
     def _timeout_check(self):
         while self.connected or self.connecting:
             if (datetime.now() - self.last_ping_time).seconds > int(self.config.core.timeout):
-                stderr('Ping timeout reached after %s seconds, closing connection' % self.config.core.timeout)
+                LOGGER.warning(
+                    'Ping timeout reached after %s seconds, '
```

Comment:
Another old grammatical mess, but less bad (because it isn't accompanied by a spelling error).

```suggestion
                    'Ping timeout reached after %s seconds; '
```